### PR TITLE
Make lead_capture_list_name a required field with default = 'Master_d…

### PR DESCRIPTION
**Pull request by reillz10**
Make lead_capture_list_name a required field with default = 'Master_default'

### CODE REVIEW

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -1373,8 +1373,7 @@ class Degree(Program):
     lead_capture_list_name = models.CharField(
         help_text=_('The sailthru email list name to capture leads'),
         max_length=255,
-        blank=True,
-        null=True,
+        default='Master_default',
```
Comment (schenedx) [inline course_discovery/apps/course_metadata/models.py:1376]: Is there a way to update the field without specifying a default? I am curious. This do not block merge and deploy.

**Review by schenedx — APPROVED** [review]
👍

File: course_discovery/apps/course_metadata/models.py (hunk 1 above)
Comment (reillz10) [reply course_discovery/apps/course_metadata/models.py:1376]: I just checked on my local machine and we DO NOT need to specify a default value. That said, I thought you mentioned that we wanted to.

Comment (schenedx) [reply course_discovery/apps/course_metadata/models.py:1376]: Let's specify one to be safe. What you have here is good to ship

**Review by iloveagent57 — APPROVED** [review]

**Pull request merged**